src/linear_algebra/basic.lean

```diff
@@ -208,8 +208,12 @@ instance : has_mul (M →ₗ[R] M) := ⟨linear_map.comp⟩
 
 lemma mul_eq_comp (f g : M →ₗ[R] M) : f * g = f.comp g := rfl
 
-@[simp] lemma one_app (x : M) : (1 : M →ₗ[R] M) x = x := rfl
-@[simp] lemma mul_app (A B : M →ₗ[R] M) (x : M) : (A * B) x = A (B x) := rfl
+@[simp] lemma one_apply (x : M) : (1 : M →ₗ[R] M) x = x := rfl
+@[simp] lemma mul_apply (A B : M →ₗ[R] M) (x : M) : (A * B) x = A (B x) := rfl
+
+@[simp] lemma coe_one : ((1 : M →ₗ[R] M) : M → M) = id := rfl
+@[simp] lemma coe_mul (A B : M →ₗ[R] M) :
+  ((A * B : M →ₗ[R] M) : M → M) = (A : M → M) ∘ (B : M → M) := rfl
```

Comment:
I'm surprised that the simp linter does't complain about these duplicating each other.

To match the rest of the file:

```suggestion
@[simp] lemma one_apply (x : M) : (1 : M →ₗ[R] M) x = x := rfl
@[simp] lemma mul_apply (f g : M →ₗ[R] M) (x : M) : (f * g) x = f (g x) := rfl

@[simp] lemma coe_one : ((1 : M →ₗ[R] M) : M → M) = id := rfl
@[simp] lemma coe_mul (f g : M →ₗ[R] M) :
  ((f * g : M →ₗ[R] M) : M → M) = (f : M → M) ∘ (g : M → M) := rfl
```

Reply:
Ah, the simp linter does indeed complain - I've pushed a fix to remove `simp` from the coe lemmas, since you were worried about those.